ArjunaJTS/interop/pom.xml

```diff
@@ -51,6 +52,43 @@
     </repository>
   </repositories>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.jboss.spec</groupId>
+      <artifactId>jboss-javaee-7.0</artifactId>
+      <version>1.1.1.Final</version>
+      <type>pom</type>
+      <scope>import</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.spec</groupId>
+      <artifactId>jboss-javaee-8.0</artifactId>
```

Comment:
Why there are needed to be the spec for ee7 and for ee8 in one pom?

Reply:
I did this change few days back, maybe I was experimenting something and forgot to remove this, it is not required. I will remove it.